### ISSUE TITLE
Cannot Progress "Signed in Blood" with "An Explorer's Footsteps" Active

### ISSUE THREAD

**Issue by Blex64** [EdenServer/community#744]
See title. Both quests use the Selbina Mayor, but it seems that because I have An Explorer's Footsteps active I cannot complete Signed in Blood. The Mayor will not give me the dialogue to progress Signed in Blood, even though I have the appropriate Key Item.

**Comment by pksweets** [comment]
On a related note, I have completed An Explorer's Footsteps to the point where I have received the Map of the Crawlers' Nest, but Abelard still asks me to retrieve monuments from zones that I have obviously already cleared instead of giving me the text I need to progress in Signed in Blood.
![leku_2019 01 27_214839](https://user-images.githubusercontent.com/27723566/51816967-b980e780-227d-11e9-9ba4-cf138624b2e3.png)

**Comment by trollsc** [comment]
What was outcome of this?

**Comment by pksweets** [comment]
As of yet unresolved. I haven't been able to get the two bonus clay tablets from Sauromugue and Batallia yet, and last I checked Abelard still gives me a recommendation to go get a tablet I've already delivered.

**Comment by pksweets** [comment]
Strangely, another of my characters, Mixolydia, was able to progress with both quests active.

**Comment by godmode** [comment]
This will be fixed on the next patch. However characters that have already started it may be stuck. If this is the case **after the patch** place a GM ticket and I will progress you to where you should be.

**Comment by reynierk** [comment]
Hi, I just wanted to clear something up because I submitted a GM tix and was told to submit a Bug Report.

I have completed "An Explorer's footsteps" to the point where I have the map. I don't care about continuing that. I care about "Signed in Blood", where I traded the "Cathedral Tapestry" to Sobane, it remained in my inventory, tried this several times to no avail. Went to speak with Mayor Abelard, and he won't tell me "Signed in Blood" dialogue, only to go get another clay tablet that I already delivered. I would like to proceed with "Signed In Blood"

I looking for advice on best approach, as I probably misunderstood the latest Patch. I submitted a GM tix, but was referred to here... 

After reading comments in this ticket and other tix, it **sounds** like I need to submit a GM Tix again... I submitted GM tix after the 2.25.19 patch
